op-scim-bridge/chart/op-scim-bridge/templates/deployer-cleanup.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: deployer-cleanup # deletes stale deployers to improve upgradeability of the SCIM bridge
```

Don't know if we want multiple of this one 

```suggestion
  name: deployer-cleanup # deletes stale deployers to improve upgradeability of the SCIM bridge
```